lib/statsd/statsdAggregator.js

```diff
@@ -27,16 +27,26 @@ StatsdAggregator.prototype._timeoutMetric = function(metric) {
     var _this = this;
 
     setTimeout(function() {
-        // fire increment
-        _this.statdsClient.increment(metric.metric, metric.value);
+        // This condition is because StatsD client for 0 delta fallbacks to increment by 1
+        if(metric.value) {
+            // fire increment
+            _this.statdsClient.increment(metric.metric, metric.value);
+        }
         _this.metrics = _.without(_this.metrics, metric);
     }, this.timeout);
 };
 
-StatsdAggregator.prototype.increment = function(metricName) {
+StatsdAggregator.prototype.increment = function(metricName, delta) {
     var metric = this._getMetric(metricName);
 
-    metric.value++;
+    if(typeof(delta) === "undefined") {
+        // Default delta value is increment by 1
+        metric.value++;
+    }
+    else {
+        // If delta value is provided then increment by delta
+        metric.value += Number.parseInt(delta);
```

Comment:
The global [`parseInt()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) would work too without requiring es2015